tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot

```diff
@@ -110,6 +110,13 @@ Get Container Logs
     ${vmName}=  Get VM Display Name  ${container}
     Should Contain  ${output}  ${vmName}/vmware.log
     Should Contain  ${output}  ${vmName}/tether.debug
+    ${rc}  ${output}=  Run And Return Rc and Output  curl -sk %{VIC-ADMIN}/container-logs.tar.gz -b /tmp/cookies-%{VCH-NAME} | (cd /tmp; tar xvzf - ${vmName}/tether.debug)
+    Log  ${output}
+    ${rc}  ${output}=  Run And Return Rc and Output  ls -l /tmp/${vm_name}/tether.debug
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc and Output  grep 'prepping for switch to container filesystem' /tmp/${vmName}/tether.debug
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc and Output  rm -f /tmp/${vmName}/tether.debug
```

Comment:
If you don't need to check ${rc} and ${output}, maybe just use `Run  rm -f ...`. This is minor though :)